optaplanner-score/pom.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <groupId>org.optaplanner</groupId>
+    <artifactId>optaplanner-build-parent</artifactId>
+    <version>8.17.0-SNAPSHOT</version>
+    <relativePath>../build/optaplanner-build-parent/pom.xml</relativePath>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>optaplanner-score</artifactId>
+  <packaging>pom</packaging>
```

Comment:
name, description, url element

At this point, you're probably wondering: why the url element? That gets inherited, no? Well the problem is that maven - because the maven-site plugin that nobody uses any more - appends the url's in inherited poms with the module name...